hooks/post_gen.dart

```diff
@@ -0,0 +1,176 @@
+import 'dart:io';
+
+import 'package:mason/mason.dart';
+
+Future<void> run(HookContext context) async {
+  _runFlutterPubGet(context);
+
+  _runAmplifyInit(context);
+
+  _runAmplifyAddAuth(context);
+
+  _runAmplifyAddApi(context);
+
+  _runAmplifyCodegen(context);
+
+  stdout.writeln("Do you want to push your changes to the cloud now?");
+  stdout.write(
+    "This might take couple of minutes (between 5-10 minutes) and you can do it any time by running 'amplify push': (Y/n)",
+  );
+  final input = stdin.readLineSync() ?? '';
+
+  _runAmplifyPush(context, input);
+}
+
+void _runFlutterPubGet(HookContext context) {
+  final flutterPubGetProgress = context.logger.progress(
+    'running "flutter pub get"',
+  );
+  final result = Process.runSync(
```

Comment:
Thoughts on using Process.start for consistency?

Reply:
Makes sense 👍🏼